doc/BorderRouter.rst

```diff
@@ -1,182 +1,220 @@
 In case a packet is identified to belong to the slow path but the queue of the slow path is full, the
 packet is dropped.
 Packets currently identified for slow-path are:
 
-* Malformed packets
+- Malformed packets
 
-* SCMP traceroute packets
+- SCMP traceroute packets
 
-## Configuration
+Configuration
+---------------
 
 The configuration of the border router will remain in the border router toml file.
 The following configuration entries are added:
 
-### Pool size
+Pool size
+^^^^^^^^^^^
 
-Since a pool of packet buffers is bound to a receiver, the size of the pool can be configured for each
-receiver seperately.
-It makes sense to make this configurable on a per border router interface level because not every
-interface might have the same expected load and hence not every interface requires the same pool size.
+The size of the packet buffer pool of the receivers can be configured.
 An optimal value should be derivable from the maximum latency of processing and forwarding a packet,
 the speed of the network interfaces and the number of available CPU cores.
 In the future we might add the possibility to automatically estimate this number.
 
-### Number of processing routines (N)
+Number of processing routines (N)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 By configuring the number of processing routines one can specify the number of goroutines that are able
 to process packets in parallel.
 An optimal value should be derivable from the maximum latency of processing and forwarding a packet,
 the speed of the network interfaces and the number of available CPU cores.
 
-### Number of slow-path processing routines (M)
+Number of slow-path processing routines (M)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 By configuring the number of slow-path processing routines one can specify the number of goroutines that
 process the packets on the slow-path.
 An optimal value could be a percentage of the number of processing routines or even a fixed number.
-In the future we might add the possibility to automatically estimate this number.
-
-### Processing packets channel size
-
-Since each processing routine has a queue of packets to process and all packets not fitting in the queue
-are dropped, one has to specify a reasonable queue size.
-A default value could be 1024.
+A default value would be 1.
 
-## Considerations for future work
+Considerations for future work
+--------------------------------
 
-### Multiple receivers per border router interface
+Multiple receivers per border router interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 We could deploy multiple packet receivers per border router interface and use eBPF to make sure that
 all packets that belong to the same flow are received by the same receiver.
 Because the rest remains unchanged we would still have the "no-reordering" guarantee and significantly
 increase the read speed.
 
-### Lock goroutines to threads
```

Comment:
We was this completely removed?

Reply:
I readded the first part of the sentence.